ibm/flex/structures.go

```diff
@@ -584,6 +584,62 @@ func FlattenIcdGroups(grouplist icdv4.GroupList) []map[string]interface{} {
 	return groups
 }
 
+func FlattenIcdGroupsV5(groupResponse *clouddatabasesv5.ListDeploymentScalingGroupsResponse) []map[string]interface{} {
```

Comment:
Can you replace `FlattenIcdGroups` with this and update the ibm_database resource to use this as well (and remove V5 suffix)